### PR TITLE
Exec copy may 11 2022

### PR DESCRIPTION
**Pull request by prose11**
Adding Exec Copy, sorry for the delay @blimpa

### CODE REVIEW

File: governance/votes/Executive vote - May 11, 2022.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: Template - [Executive Vote] Recognized Delegate Payments, Lowering WBTC Stability Fees, and Onboarding wstETH-B - May 11, 2022
+summary: Distributing 79,647 DAI to Recognized Delegates, Lowering WBTC-A and WBTC-B Stability Fees, and onboarding wstETH-B (Wrapped Lido Staked ETH).
```
Comment (blimpa) [inline governance/votes/Executive vote - May 11, 2022.md:3]: Inconsistent capitalization

File: governance/votes/Executive vote - May 11, 2022.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+* 8,626 DAI transferred to [monetsupply](https://vote.makerdao.com/address/0x45127ec92b58c3a89e89f63553073adcaf2f1f5f) at [0x4Bd73eeE3d0568Bb7C52DFCad7AD5d47Fff5E2CF](https://etherscan.io/address/0x4Bd73eeE3d0568Bb7C52DFCad7AD5d47Fff5E2CF).
+* 7,522 DAI transferred to [JustinCase](https://vote.makerdao.com/address/0xcdb792c14391f7115ba77a7cd27f724fc9ea2091) at [0xE070c2dCfcf6C6409202A8a210f71D51dbAe9473](https://etherscan.io/address/0xE070c2dCfcf6C6409202A8a210f71D51dbAe9473).
+* 6,607 DAI transferred to [GFX Labs](https://vote.makerdao.com/address/0xf60d7a62c98f65480725255e831de531efe3fe14) at [0xa6e8772af29b29B9202a073f8E36f447689BEef6](https://etherscan.io/address/0xa6e8772af29b29B9202a073f8E36f447689BEef6).
+* 351 DAI transferred to [Doo](https://vote.makerdao.com/address/0x8804d391472126da56b9a560aef6c6d5aaa7607b#delegate-credentials) at [0x3B91eBDfBC4B78d778f62632a4004804AC5d2DB0](https://etherscan.io/address/0x3B91eBDfBC4B78d778f62632a4004804AC5d2DB0).
```
Comment (blimpa) [inline governance/votes/Executive vote - May 11, 2022.md:49]: Remove anchor (...#delegate-credentials)

File: governance/votes/Executive vote - May 11, 2022.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+* 265 DAI transfered to [Flipside Crypto](https://vote.makerdao.com/address/0x84b05b0a30b6ae620f393d1037f217e607ad1b96) at [0x62a43123FE71f9764f26554b3F5017627996816a](https://etherscan.io/address/0x62a43123FE71f9764f26554b3F5017627996816a).
+
+### Lower WBTC-A and WBTC-B Stability Fee
```
Comment (blimpa) [inline governance/votes/Executive vote - May 11, 2022.md:52]: We used plural in the title ("Fees") :shrug:

File: governance/votes/Executive vote - May 11, 2022.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+* Lower the WBTC-A [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) from 3.25% to **2.25%**.
+* Lower the WBTC-B [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) from 4.5% to **4.0%**.
+
+### Onboard wstETH-B as a New Vault Type
+
+As per this [governance poll](https://vote.makerdao.com/polling/QmaE5doB#poll-detail), following this [recomendation](https://forum.makerdao.com/t/introduction-risk-parameters-for-wsteth-c-vault/14669) from the [Risk Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP2), wstETH-B will be onboarded to the Maker Protocol with the following parameters, if this executive proposal passes: 
+
+*Note that this proposed vault type was previously discussed as wstETH-C, however, if it is voted to be added to the Maker Protocol it will be added under the wstETH-B ilk name to maintain sequential naming conventions as discussed [here](https://forum.makerdao.com/t/ilk-names-versus-display-names-for-vault-types/14759).*
```
Comment (blimpa) [inline governance/votes/Executive vote - May 11, 2022.md:63]: Should "vault type" be capitalized?

File: governance/votes/Executive vote - May 11, 2022.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+### Initial Vault Parameters
```
Comment (blimpa) [inline governance/votes/Executive vote - May 11, 2022.md:65]: This should be either H4 (####) or bold

File: governance/votes/Executive vote - May 11, 2022.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+* Underlying Collateral: wstETH (Wrapped Lido Staked ETH)
+* [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee): 0.75%
+* [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio): 185%
+* [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line): 150 million DAI
+* [Debt Floor (`dust`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-floor): 5,000 DAI
+* [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap): 15 million DAI
+* [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl): 8 hours
+
+### Liquidation Parameters
```
Comment (blimpa) [inline governance/votes/Executive vote - May 11, 2022.md:75]: This should be either H4 (####) or bold

File: governance/votes/Executive vote - May 11, 2022.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+* [Auction Price Function (`calc`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-auction-price-function): Exponential Stair Step
+* [Price Change Multiplier (`cut`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-auction-price-function#exponential-stair-step): 0.99
+* [Price Change Interval (`step`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-auction-price-function#exponential-stair-step): 90 seconds
+* [Auction Price Multiplier (`buf`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-auction-price-multiplier): 1.2
+* [Liquidation Penalty (`chop`)](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-penalty): 13%
+
+**Limits**
```
Comment (blimpa) [inline governance/votes/Executive vote - May 11, 2022.md:83]: H4 (####)?

File: governance/votes/Executive vote - May 11, 2022.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+* [Local Liquidation Limit (`ilk.hole`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-local-liquidation-limit): 10 million DAI
+* [Maximum Auction Drawdown (`cusp`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-max-auction-drawdown): 0.4
+* [Maximum Auction Duration (`tail`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-max-auction-duration): 140 minutes
+* [Breaker Price Tolerance (`tolerance`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-breaker-price-tolerance): 0.5
+
+**Incentives**
```
Comment (blimpa) [inline governance/votes/Executive vote - May 11, 2022.md:90]: H4 (####)?

**Review by blimpa — CHANGES_REQUESTED** [review]
Minor formatting issues :)

File: governance/votes/Executive vote - May 11, 2022.md (hunk 3 above)
Comment (prose11) [reply governance/votes/Executive vote - May 11, 2022.md:52]: I fee, two fee, three fee, floor

**Comment by prose11** [comment]
Thanks for the corrections @blimpa merging through

**Pull request merged**